### PR TITLE
Fix linters error

### PR DESCRIPTION
**Pull request by stanleySimeon**
# In this milestone I built a fully functional mobile web app using ruby on rails.
  

  - This app allows users to have control over their spend
  - users can add a lot of categories of payments
  - users can manage each payment
  - users are able to create an account (login and sign in)
 
- This Project was built with:
  - `Ruby on rails`
  - `Tailwind css`
 
- Testing with:
  - `Capybara`
  - `Rspec`

Screen Shot 2022-09-16 at 22 50 14

Screen Shot 2022-09-16 at 22 49 56

Screen Shot 2022-09-16 at 00 47 17

### CODE REVIEW

File: spec/requests/categories_spec.rb [hunk 1]
```diff
@@ -8,18 +8,17 @@
...
-  #     get '/categories/new'
-  #     expect(response).to have_http_status(:success)
-  #   end
-  # end
+# describe 'GET /new' do
+#   it 'returns http success' do
+#     get '/categories/new'
+#     expect(response).to have_http_status(:success)
+#   end
+# end
```
Comment (MahmoudBakr23) [inline spec/requests/categories_spec.rb]: - [x] Kindly uncomment the tests' blocks of code and refactor them to accept test cases according to the project requirements to have units and integration tests. Please do so in all of the other related files! 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @stanleySimeon 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done designing the app
- Great work with Restful methods in the controllers 
- Well done setting up the models' associations and validations according to the ERD
- Great job working with the corresponding views to show and interact with the records
- Well done with the migrations and database schema
- Nicely done with the devise gem for the authentication
- Descriptive PR and correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process improves the code you wrote, not criticizes it. 😄

 ## Required Changes ♻️

- [x] Please make sure to record a 5 min presentation video about your app
- [x] Kindly make sure to attach the deployed version link into the README file

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by stanleySimeon** [comment]
Hello @MahmoudBakr23, I can record only 5 mins video on loom.
Kindly, you can tell me how it should equal to 5mins?

File: spec/requests/categories_spec.rb (hunk 1 above)
Comment (stanleySimeon) [reply spec/requests/categories_spec.rb]: I'll do that!

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @stanleySimeon 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process improves the code you wrote, not criticizes it. 😄

 ## Required Changes ♻️

- [ ] Please make sure to deploy the app and attach the live demo link in the README file
- [ ] Regarding your comment about loom, please note that the presentation video could be 3 or 4 mins as long as it covers the features of the app in a descriptive way. The important thing is that you don't exceed 5 mins, and that's what loom offers. 😄 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by stanleySimeon** [comment]
@MahmoudBakr23 the video is exactly 4mins
Kindly, where is the problem to approve the project?

**Comment by MahmoudBakr23** [comment]
Hi @stanleySimeon 
As we discussed on Slack, Please attach the link to your video and the link to your live demo link in order to get the approval and avoid repeating the week! 🙏 
Thank you for contacting me

**Comment by AshakaE** [comment]
Hi @stanleySimeon , 

### Highlights ✨ 

Thanks for making the requested changes 

- Code base looks clean 👍 
- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**